site/src/pages/tutorials/grpc/blog/index.mdx

```diff
@@ -9,7 +9,7 @@ import versions from '/gen-src/versions.json';
 
 Learn to build a gRPC service with Armeria by writing a blog service in gRPC. This tutorial is based on a [sample service](https://github.com/line/armeria-examples/tree/main/tutorials/grpc), a minimal blog service, with which you can create, read, update, and delete blog posts.
 
-Before we get our hands on the tutorial, let's switfly go over Armeria's gRPC features. Armeria supports features that the upstream (gRPC-Java) does not support.
+Before we get our hands on the tutorial, let's swiftly go over Armeria's gRPC features. Armeria supports features that the upstream (gRPC-Java) does not support.
```

Comment:
This will be fixed by https://github.com/line/armeria/pull/5445/files so let's revert this. 😉